purchases/src/main/kotlin/com/revenuecat/purchases/models/PricingPhase.kt

```diff
@@ -54,14 +56,50 @@ data class PricingPhase(
             }
         }
 
+    /**
+     * Gives the price of the [PricingPhase] in the given locale in a weekly recurrence. This means that for example,
+     * if the period is monthly, the price will be divided by 4. It uses a currency formatter to format the price in
+     * the given locale. Note that this value may be an approximation.
+     * @param locale Locale to use for formatting the price. Default is the system default locale.
+     */
+    @JvmOverloads
+    fun pricePerWeek(locale: Locale = Locale.getDefault()): Price =
+        price.pricePerWeek(billingPeriod, locale)
+
+    /**
+     * Gives the price of the [PricingPhase] in the given locale in a monthly recurrence. This means that for example,
+     * if the period is annual, the price will be divided by 12. It uses a currency formatter to format the price in
+     * the given locale. Note that this value may be an approximation.
+     * @param locale Locale to use for formatting the price. Default is the system default locale.
+     */
+    @JvmOverloads
+    fun pricePerMonth(locale: Locale = Locale.getDefault()): Price =
+        price.pricePerMonth(billingPeriod, locale)
+
+    /**
+     * Gives the price of the [PricingPhase] in the given locale in a yearly recurrence. This means that for example,
+     * if the period is monthly, the price will be multiplied by 12. It uses a currency formatter to format the price in
+     * the given locale. Note that this value may be an approximation.
+     * @param locale Locale to use for formatting the price. Default is the system default locale.
+     */
+    @JvmOverloads
+    fun pricePerYear(locale: Locale = Locale.getDefault()): Price =
+        price.pricePerYear(billingPeriod, locale)
+
     /**
      * Gives the price of the [PricingPhase] in the given locale in a monthly recurrence. This means that for example,
      * if the period is annual, the price will be divided by 12. It uses a currency formatter to format the price in
      * the given locale. Note that this value may be an approximation.
+     *
+     * This is equivalent to:
+     * ```kotlin
+     * pricePerMonth(locale).formatted
+     * ```
+     *
      * @param locale Locale to use for formatting the price. Default is the system default locale.
      */
     @JvmOverloads
     fun formattedPriceInMonths(locale: Locale = Locale.getDefault()): String {
-        return price.pricePerMonth(billingPeriod, locale).formatted
+        return pricePerMonth(locale).formatted
```

Comment:
Should we deprecate this and direct people to use `pricePerMonth(locale).formatted` directly? What do you think? It might be confusing that 2 methods do pretty much the same thing.

Reply:
Yeah, I agree with that. `formattedPriceInMonths` doesn't really fit the naming we've been using 👍

Reply:
Alright, done: https://github.com/RevenueCat/purchases-android/pull/1789/commits/5947b8b4399ee83b0e1e5c2a5fbfc0ae4456d1d3